databroker/tests/test_broker.py

```diff
@@ -242,3 +242,22 @@ def test_raise_conditions():
     ]
     for raiser in raising_keys:
         yield _raiser_helper, raiser
+
+
+def test_stream():
+    s = stream(db[-1])
```

Comment:
There is a module level `setup()` function that gets run first, so the broker will always have access to a header